.changeset/silly-eggs-cross.md

```diff
@@ -0,0 +1,5 @@
+---
+"svelte": patch
+---
+
+fix: improve each keyed block handling of item used as key
```

Comment:
I think at this point it's ok to not have a changeset, we already have two related to this and haven't released it yet 😄